core/mate/model/src/main/java/org/opennms/core/mate/model/EntityScopeProviderImpl.java

```diff
@@ -26,12 +26,7 @@
  *     http://www.opennms.com/
  *******************************************************************************/
 
-package org.opennms.core.rpc.utils.mate;
-
-import static org.opennms.core.rpc.utils.mate.EntityScopeProvider.Contexts.ASSET;
-import static org.opennms.core.rpc.utils.mate.EntityScopeProvider.Contexts.INTERFACE;
-import static org.opennms.core.rpc.utils.mate.EntityScopeProvider.Contexts.NODE;
-import static org.opennms.core.rpc.utils.mate.EntityScopeProvider.Contexts.SERVICE;
```

Comment:
Why switching from static imports to explicit references? Did anything else change on this file?

Reply:
done.